modules/ghlint-snakeyaml/src/test/kotlin/net/twisterrob/ghlint/model/SnakePermissionsTest.kt

```diff
@@ -0,0 +1,151 @@
+package net.twisterrob.ghlint.model
+
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import net.twisterrob.ghlint.testing.load
+import net.twisterrob.ghlint.testing.loadUnsafe
+import org.junit.jupiter.api.Test
+
+class SnakePermissionsTest {
+
+	private fun File.asJob(): Job.NormalJob = (content as Workflow).jobs.values.single() as Job.NormalJob
```

Comment:
```suggestion
	private val File.theJob: Job.NormalJob = (content as Workflow).jobs.values.single() as Job.NormalJob
```
(as is a keyword and it means casting)